CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ _ `_optional` subpackage for managing optional dependencies
 - Abstract `CardinalityConstraint` class and
   `DiscreteCardinalityConstraint`/`ContinuousCardinalityConstraint` subclasses
 - Uniform sampling mechanism for continuous spaces with cardinality constraints
+- Enable `ContinuousCardinalityConstraint` in `BotorchRecommender`
```

Comment:
@Waschenbacher @AdrianSosic what is the current working mode? Do you want to do a first round of reviews without intervention from me and @Scienfitz or should we already review this (and consequently each future PR made by @Waschenbacher ) directly?

Reply:
Let me do the first round, since I've already had several technical discussions with @Waschenbacher the last days about it. Once the basic layout is OK, I'll ping you 👍🏼